java/ingest/ingest-status-store/src/test/java/sleeper/ingest/status/store/job/QueryIngestJobStatusByPeriodIT.java

```diff
@@ -86,18 +86,16 @@ public void shouldExcludeIngestJobInOtherTable() {
     }
 
     @Test
-    public void shouldIncludeFinishedStatusUpdateOutsidePeriod() throws Exception {
+    public void shouldIncludeFinishedStatusUpdateOutsidePeriod() {
         // Given
         IngestJob job = jobWithFiles("file");
-        Instant periodStart = Instant.now().minus(Period.ofDays(1));
-        Instant startedTime = Instant.now();
+        Instant startedTime = Instant.parse("2023-01-03T14:50:00.001Z");
+        Instant periodStart = startedTime.minus(Period.ofDays(1));
```

Comment:
If we state the full time explicitly with Instant.parse, we can state all the times in the order they happen. The times declared below can also be moved up here.